test/block_test.rb

```diff
@@ -0,0 +1,91 @@
+require_relative 'test_helper'
+
+describe "Block" do
+  
+  before do
+    @hotel = Hotel::ReservationManager.new
+    
+    @test_rooms = []
+    5.times do |i|
+      @test_rooms << @hotel.rooms[i]
+    end
+    
+    @block = Hotel::Block.new(Date.parse('2019-10-31'), Date.parse('2019-11-04'), @test_rooms, discount: 0.8)    
+    @hotel.blocks << @block
+  end
+  
+  it "initializes a Block" do
+    expect(@block).must_be_instance_of Hotel::Block
+  end
+  
+  it "includes all dates from checkin to checkout in a Block" do
+    expect(@block.dates).must_be_instance_of Array
+    expect(@block.dates[1].to_s).must_equal '2019-11-01' 
+    expect(@block.dates.length).must_equal 4 
+  end
+  
+  it "adds up to 5 rooms to a Block" do
+    new_block = @hotel.request_reservation(Date.parse('2019-10-31'), Date.parse('2019-11-04'), type: :block, amount: 5)
```

Comment:
Because the "Act" step is a method that belongs to a `ReservationManager` (aka the method `request_reservation`), this test should be moved to the `reservation_manager_test`